src/System.Console/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -1,2 +1,28 @@
-<!-- Not a bug, this API was recently added to the allowed list -->
-user32.dll!GetKeyState
+kernel32.dll!Beep
+kernel32.dll!FillConsoleOutputAttribute
+kernel32.dll!FillConsoleOutputCharacterW
```

Comment:
Are we building System.Console the right way that we are getting all these violations?

There is stubbed out version of Console for UWP (https://github.com/dotnet/corefx/blob/master/src/System.Console/src/System/ConsolePal.Uap.cs) that does not PInvoke to anything, but it does not seem to be hooked up in the build correctly.